scripts/nuke.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# This script removes all transient directories (node_modules and build results)
+# To only be used after you've stopped all servers
+
+NUKE_DIRS="node_modules unlock-app/node_modules/ locksmith/node_modules/ tests/node_modules/ smart-contracts/node_modules/ unlock-app/build unlock-app/coverage unlock-app/src/out unlock-app/src/.next unlock-app/coverage unlock-app/src/out unlock-app/src/.next smart-contracts/build/contracts/*.json"
+
+# First check if ganache is running
+GANACHE_PORT=8545
+lsof -i :$GANACHE_PORT >> /dev/null
+if [ $? -eq 0 ]; then
+  echo "Please make sure you stop ganache first"
+  exit 1
```

Comment:
What's the best way to do this? Sometimes I seem to get an "orphaned" ganache process running in the background, even when it seems I've stopped ganache. I've been using `alias kill8545="lsof -t -i tcp:8545 | xargs kill"` in my `.bashrc`

Reply:
That should work. I did not want to normalize because this may be very much OS specific.